.travis.yml

```diff
@@ -14,11 +16,16 @@ before_install:
   - cp config/secrets.travis.yml config/secrets.yml
   - git submodule init
   - git submodule update --depth=10 -- red5
+  - rvm use 2.3.3@global
+  # Bundler 1.15 has stricter dependency checking -- problematic for Hydra projects
+  - gem uninstall -aIx bundler
+  - gem install bundler -v 1.14.6
   - bundle config without development:production:staging
 before_script:
   - rm config/database.yml
   - cp config/database.travis.yml config/database.yml
   - mysql -e 'create database avalon_test;'
```

Comment:
Any reason not to just do this through rails/rake? `rake db:create` etc? Or better yet in most rails projects now days they have a binstub for setting up your application called `bin/setup`